### DISCUSSION TITLE
Support ninths

### DISCUSSION THREAD

**Discussion by dr-eme** [rxhanson/Rectangle#396] [Ideas]
For big 16:9 screen owners (4K,5K,6K), it would be awesome to resize windows in ninths, and maybe have the default shortcuts to the numeric keypad:

|  7  |  8  |  9  |
|  4  |  5  |  6  |
|  1  |  2  |  3  |

**Comment by rxhanson** [comment]
Thanks for the idea! I don't have plans to add this into Rectangle at this time, although I would consider merging a pull request for it.

There are so many desired edge cases for window placement and dimensions, that I did build a custom shortcut builder for my paid superset of Rectangle, Hookshot (https://hookshot.app). The only thing that Hookshot would not be able to do for you is allow the shortcuts to be executed using just the keypad - they would still need a modifier key. Here's a post I wrote about how to use the custom shortcut builder (and why it's not a part of Rectangle): https://medium.com/ryan-hanson/custom-window-dimension-shortcuts-in-macos-e7df347dfd24

**Comment by slifty** [comment]
Just wanted to voice a second for this concept -- having three stacked terminals that each take up a ninth is a common situation for me and I've found to be very valuable via Divvy.

**Reply by slifty** [reply]
I opened a PR for this feature: https://github.com/rxhanson/Rectangle/pull/702
